tests/Doctrine/Tests/ORM/Functional/PersistentCollectionTest.php

```diff
@@ -95,6 +95,28 @@ public function testMatchingDoesNotModifyTheGivenCriteria()
         $this->assertEmpty($criteria->getMaxResults());
         $this->assertEmpty($criteria->getOrderings());
     }
+
+    public function testPersistClearCollection()
+    {
+        $collectionHolder = new PersistentCollectionHolder();
+        $content = new PersistentCollectionContent('first element');
+        $collectionHolder->addElement($content);
+
+        $this->_em->persist($collectionHolder);
+        $this->_em->flush();
+        $this->_em->clear();
+
+        $collectionHolder = $this->_em->find(PersistentCollectionHolder::class, $collectionHolder->getId());
+        $this->assertCount(1, $collectionHolder->getCollection());
```

Comment:
This is a static method, so please use `self::assertCount()` instead of `$this->assertCount()`